packages/profiles/source/contact-address.repository.test.ts

```diff
@@ -154,7 +146,6 @@ void describeWithContext(
 				context.subject.exists({
 					address: "DAWdHfDFEvvu57cHjAhs5K5di33B2DdCu1",
```

Comment:
will add a card to have a look at these addresses being used to change them to `keccak` ones instead for consistency